application-admintools-ui/src/main/resources/AdminTools/Code/HealthCheckResources.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+<xwikidoc version="1.4" reference="AdminTools.Code.HealthCheckResources" locale="">
+  <web>AdminTools.Code</web>
+  <name>HealthCheckResources</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>Main.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>HealthCheckResources</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{template name="job_macros.vm"/}}
+
+{{velocity output="false"}}
+#macro (returnHealthCheckJobStatusJSON $healthCheckJobId)
+  #set ($healthCheckJobStatus = $services.job.getJobStatus($healthCheckJobId))
+  #set ($healthCheckJobStatusJSON = {
+    'id': $healthCheckJobStatus.request.id,
+    'state': $healthCheckJobStatus.state,
+    'progress': {
+      'offset': $healthCheckJobStatus.progress.offset
+    }
+  })
+  #jsonResponse($healthCheckJobStatusJSON)
+#end
+
+#macro (getHealthCheckResources)
+  #set ($jobMinJsURL = $xwiki.getSkinFile('uicomponents/job/job.min.js'))
+  #set ($healthCheckJobRequest = $services.admintools.createJobRequest($services.wiki.getCurrentWikiId()))
+  #set ($healthJobStatus = $services.job.getJobStatus($healthCheckJobRequest.id))
+  {{html clean=false}}
+   &lt;div hidden>
+     &lt;input type="text" id="jobMinJsURL" name="jobMinJsURL" value=$jobMinJsURL>&lt;/input>
```

Comment:
It's not clear to me why you need this, but:
* the value needs to be quoted and XML escaped
* in general, we use input elements to pass data from the client to the server; in this case you're passing data from the server to the client, which can be done either using ``data-*`` attributes (for simple data) or using a script tag with JSON content, for complex data.